lefthook.yml

```diff
@@ -0,0 +1,9 @@
+pre-commit:
+  parallel: true
+  commands:
+    prettier:
+      # run: pnpm run prettier:check
+      description: "Check for prettier formatting"
+      run: npx prettier --check {staged_files} --ignore-path=.prettierignore
```

Comment:
should this be using `pnpm` ?

Reply:
Good catch. Updated in 929f32be967b63aa49e24ed7b3bcb81240a346b9.